src/common/reduction/matching/GapMatcher.h

```diff
@@ -26,6 +27,9 @@ class GapMatcher : public AbstractMatcher {
   ///         the clusters are merged into one event.
   void set_minimum_time_gap(uint64_t minimum_time_gap);
 
+  void set_split_multi_events(bool split_multi_events, float coefficient_low,
```

Comment:
All methods specified in .h files should have Doxygen comments describing what they to, what
parameters they take and, if relevant, the returned data.

I also forget to do this so we should keep reminding each other about this